### PR TITLE
update dictionaries

### PR DESCRIPTION
**Pull request by kasparvonbeelen**

### CODE REVIEW

File: Sessions/2e-Dictionaries-and-json.ipynb [hunk 1]
```diff
@@ -842,44 +767,61 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "235f4fea",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sorted(wordcounts.items(),key=get_second_item)"
```
Comment (mcollardanuy) [inline Sessions/2e-Dictionaries-and-json.ipynb:829]: I don't think it is obvious (with what's been taught so far) what happens here and that these two last blocks do the same, maybe a description would be needed (or make sure you mention it in class).

File: Sessions/2e-Dictionaries-and-json.ipynb [hunk 2]
```diff
@@ -1155,43 +1082,14 @@
...
-      ]
-     },
-     "execution_count": 20,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
     "word_counts = Counter(sentence)\n",
```
Comment (mcollardanuy) [inline Sessions/2e-Dictionaries-and-json.ipynb]: I don't know if this is intended, but `sentence` here is the string: so you are getting a counter of characters in the string, not a counter of tokens. This should be `word_counts = Counter(word_list)`, right?

File: Sessions/2e-Dictionaries-and-json.ipynb [hunk 3]
```diff
@@ -1517,6 +1402,40 @@
...
+    }
+   },
+   "source": [
+    "✏️ **3. Exercise:** \n",
+    "\n",
+    "\n",
+    "- Change the search term in `api_query` from 'sufrage' to something else\n",
+    "- Collect the JSON response\n",
+    "- Save all the 'ocr_text' in a list called `text`\n",
+    "- Join the text into one string with `' '.join(text)'\n",
```
Comment (mcollardanuy) [inline Sessions/2e-Dictionaries-and-json.ipynb]: Formatting typo, it should be:
```
`' '.join(text)`
```

**Review by mcollardanuy — DISMISSED** [review]
Apart from these minor comments, all good!

File: Sessions/2e-Dictionaries-and-json.ipynb (hunk 1 above)
Comment (kasparvonbeelen) [reply Sessions/2e-Dictionaries-and-json.ipynb:829]: I agree, I think it's important to stress here they can just copy paste? It's difficult to understand but I will mention it in class.

Comment (kasparvonbeelen) [reply Sessions/2e-Dictionaries-and-json.ipynb:829]: Added a docstring to the function, to make things a bit clearer

File: Sessions/2e-Dictionaries-and-json.ipynb (hunk 2 above)
Comment (kasparvonbeelen) [reply Sessions/2e-Dictionaries-and-json.ipynb]: Yes, correct! Thanks for spotting this ;-) Would make a good exercise, hehe

File: Sessions/2e-Dictionaries-and-json.ipynb (hunk 3 above)
Comment (kasparvonbeelen) [reply Sessions/2e-Dictionaries-and-json.ipynb]: fixed!

**Review by mcollardanuy — APPROVED** [review]

**Pull request merged**